### PR TITLE
Adjustments to near term trends for ARIADNE, stocks & sales fix

### PR DESCRIPTION
**Pull request by Loisel**
Indroducing a different notion of stocks and sales: Previously, we thought of stock as all cars that has been build up to the year x, sales were the cars from year x. This has been revised to stock being all cars, sales being only the cars from the current year

### CODE REVIEW

File: R/reportEDGET.R [hunk 1]
```diff
@@ -327,9 +327,12 @@ toolReportEDGET <- function(output_folder = ".",
   reportStockAndSales <- function(annual_mileage, load_factor) {
     year_c <- construction_year <- Stock <- Sales <- vintage_demand_vkm <- fct <- category <- NULL
     vint <- vintageReport(load_factor)
-    vint[, stock_demand := sum(vintage_demand_vkm), by=c("year", "region", "vehicle_type", "technology")]
-    vint[, sales_demand := full_demand_vkm - stock_demand]
-    vint <- unique(vint[, c("construction_year", "vintage_demand_vkm", "full_demand_vkm") := NULL])
+    vint[, tot_vint_demand := sum(vintage_demand_vkm), by=c("year", "region", "vehicle_type", "technology")]
```
Comment (johannah-pik) [inline R/reportEDGET.R:330]: What gets summed up here?

File: R/compareScenarios_EDGET.R [hunk 2]
```diff
@@ -151,7 +151,7 @@ compareScenarios_EDGET <- function(
     newYaml,
     path = file.path(pathDir, "cs2_main.Rmd"),
     template = system.file(
-      "rmd/compareScenarios_Transport/csEDGET_main.Rmd",
+      "Rmd/compareScenarios_Transport/csEDGET_main.Rmd",
```
Comment (johannah-pik) [inline R/compareScenarios_EDGET.R]: Sad that we spotted it so late and somehow funny then both at the same time :D Hope it works now for you to render the CompareScenarios!

File: R/reportEDGET.R (hunk 1 above)
Comment (Loisel) [reply R/reportEDGET.R:330]: the vintages from each year.

Comment (johannah-pik) [reply R/reportEDGET.R:331]: Are there different vintages for the same technology and vehicle time for each year? Or what am I missing?

File: R/reportEDGET.R [hunk 3]
```diff
@@ -327,9 +327,12 @@ toolReportEDGET <- function(output_folder = ".",
...
+    vint[, sales_demand := full_demand_vkm - tot_vint_demand]
```
[the remark this answers is not in the document]
Comment (Loisel) [reply R/reportEDGET.R:331]: I edited the description of the PR as this is not clear.

**Review by johannah-pik — APPROVED** [review]

**Comment by Loisel** [comment]
yes, sorry for merging before I answer the question I just saw the approval. The vintages have the additional dimension of the construction year which gets summed here.

**Pull request merged**